synth.py

```diff
@@ -30,7 +30,7 @@
     },
```

Comment:
In line 28 above, I'm pretty sure `synthtool` does not expect JavaScript package name as the library name.